static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js

```diff
@@ -3,91 +3,93 @@ import React, { Component } from 'preact-compat';
 import { FollowCard } from 'common/modules/identity/upsell/consent-card/FollowCard';
 import type { ConsentWithState } from '../store/types';
 import { setConsentsInApi } from '../store/consents';
+import { ErrorBar, genericErrorStr } from '../error-bar/ErrorBar';
+
+const joinWithOr = arr =>
+    arr.reduce(
+        (acc, val, idx, src) =>
+            idx === 0
+                ? val
+                : [acc, idx + 1 >= src.length ? ', or ' : ', ', val].join(''),
```

Comment:
I'd actually prefer this on one line `idx === 0 ? val : [acc, idx + 1 >= src.length ? ', or ' : ', ', val].join('')`

But that's probably just me

Reply:
Just seen that this is how it's done everywhere, so if that's the JS convention 👍

Reply:
we can't really choose the js styling, prettier will reformat it :D

I think this function could benefit from being less clever-y and shorthanded-y if legibility is an issue but also since the name is so self-descriptive it's quite easy to understand what it does and